.github/workflows/wheel_generation.yml

```diff
@@ -37,10 +38,14 @@ jobs:
       with:
         path: ${{ env.COLCON_WS }}/src/${{ env.PACKAGE_NAME }}
     # clone public dependencies
-    - name: git clone maliput
+    - name: vcs import
+      shell: bash
+      working-directory: ${{ env.COLCON_WS }}
+      run: PATH=$PATH:${{ env.VCS_PATH }} vcs import src < src/${PACKAGE_NAME}/.github/dependencies.repos
```

Comment:
Note that `dependencies.repos` is also bringing ament_cmake_doxygen which is not necessary in this workflow. When doing `colcon build` I am not sure if there will be an issue for this package as we are not running rosdep install before due to obvious reasons and we are installing the dependencies manually.